politeiawww/prices.go

```diff
@@ -118,10 +233,12 @@ func (p *politeiawww) processInvoiceExchangeRate(ier cms.InvoiceExchangeRate) (c
 		if err == database.ErrExchangeRateNotFound {
 			monthAvgRaw, err := p.GetMonthAverage(time.Month(ier.Month), int(ier.Year))
 			if err != nil {
+				log.Error(err)
```

Comment:
You should give error messages context when logging them to make them more helpful.  Example:

```
log.Errorf("processInvoiceExchangeRate: GetMonthAverage: %v", err)
```